src/drivers/mct/cime_config/config_component.xml

```diff
@@ -2026,6 +2026,14 @@
     <desc>Machines directory location</desc>
   </entry>
 
+  <entry id="EXTRA_MACHDIR">
+    <type>char</type>
+    <default_value></default_value>
+    <group>case_def</group>
+    <file>env_case.xml</file>
+    <desc>Path to an extra directory containing supplementary machines files</desc>
+  </entry>
```

Comment:
It looks like I'll need to make a similar change for cmeps/nuopc. I'll open a PR with that change soon.

Reply:
Done - see https://github.com/ESCOMP/CMEPS/pull/62